### PR TITLE
RP PR edition: More Emotes

### PR DESCRIPTION
**Pull request by Jolly-66**
Yeah I got nothing.

List of cool emotes you can now do:

- Slow clap

### CODE REVIEW

**Comment by Nikitaw99** [comment]
- *hiss
- a finger *snap
- something tail-related maybe

File: jollystation_modules/code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	vary = TRUE
+
+/datum/emote/living/carbon/slow_clap/get_sound(mob/living/user)
+	if(ishuman(user))
+		if(!user.get_bodypart(BODY_ZONE_L_ARM) || !user.get_bodypart(BODY_ZONE_R_ARM))
+			return
+		else
+			return pick('jollystation_modules/sound/misc/slowclap1.ogg',
+							'jollystation_modules/sound/misc/slowclap2.ogg',
+							'jollystation_modules/sound/misc/slowclap4.ogg')
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/mob/living/carbon/emote.dm:19]: the slow-clap already in game sounds better than these imo

**Comment by Wallemations** [comment]
I always find myself doing a dismissive wave, so *dismiss could work
like,
> "Ass Istant waves dismissively."
could also use `message_param` to make it so you wave dismissively at a specific thing

File: jollystation_modules/code/modules/mob/living/carbon/emote.dm (hunk 1 above)
Comment (MrMelbert) [reply jollystation_modules/code/modules/mob/living/carbon/emote.dm:19]: image

**Comment by Ed640** [comment]
I don't remember if there was a way to have tails intertwine or if that was a joke

**Comment by Jolly-66** [comment]
> I don't remember if there was a way to have tails intertwine or if that was a joke

This was a joke by Oranges

**Pull request closed without merging**